logstash-core/spec/logstash/filter_delegator_spec.rb

```diff
@@ -92,6 +93,7 @@ def filter(event)
       end
 
       it "increments the in/out of the metric" do
+        allow(metric).to receive(:increment).with(:time_ms, anything)
```

Comment:
This test, to me, intends to test the "in" and "out" metrics. It doesn't mention any assertion about "time_ms", so it feels correct to use `allow` here because we don't care if it's called, but we must allow it otherwise the test will fail.

Thoughts?